packages/stream_chat_flutter/lib/src/message_input.dart

```diff
@@ -1191,6 +1285,10 @@ class MessageInputState extends State<MessageInput> {
         })?.toList() ??
         [];
 
+    if (members.isEmpty) {
+      return null;
```

Comment:
you can just `return;` here, no need for an explicit `null`.

Reply:
I'm gonna leave this because in the analysis_options we are using in this package this is mandatory
We'll remove it once we apply the other one